.eslintrc.js

```diff
@@ -1,15 +1,44 @@
 module.exports = {
-    "env": {
-        "commonjs": true,
-        "es6": true,
-        "node": true
-    },
-    "extends": "eslint:recommended",
-    "globals": {
-    },
-    "parserOptions": {
-        "ecmaVersion": 2018
-    },
-    "rules": {
-    }
-};
\ No newline at end of file
+    extends: ['@polarislabs/eslint-config-polaris/node'],
+    parser: 'babel-eslint',
+    overrides: [
+        {
+            'files': ['*'],
+            'rules': {
+                'no-console': 'off'
+            }
+        },
+        {
+            'files': ['*'],
+            'rules': {
+                '@typescript-eslint/no-var-requires': 'off'
+            }
+        },
+        {
+            'files': ['*'],
+            'rules': {
+                'camelcase': 'off'
+            }
+        },
+        {
```

Comment:
Please fix these last 3 rules instead of ignoring them. There aren't many violations:
```
/Users/bwortman/Documents/Development/pg-to-dbml/src/main.js
  51:1  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions

/Users/bwortman/Documents/Development/pg-to-dbml/src/utils/createFile.js
  4:28  error  Unexpected use of '|'            no-bitwise
  5:39  error  Unexpected empty arrow function  @typescript-eslint/no-empty-function

/Users/bwortman/Documents/Development/pg-to-dbml/src/utils/writeToFile.js
  4:56  error  Unexpected empty arrow function  @typescript-eslint/no-empty-function
```